department-of-veterans-affairs/caseflow
# Display hearing time timezones on schedule veterans page

Resolves #15394 

### Description

 - Updates the `RegionalOffice` dropdown options to include the entire regional office object instead of just the RO key
   - Lots of small updates related to this for other components that use the `RegionalOffice` component
 - Pass in `localTimezone` to `HearingTime` components
 - Update tests and storybook

### Test Plan
1. Shadow user `BVASYELLOW`
1. Navigate to the schedule hearings page http://localhost:3000/hearings/schedule/assign
1. Choose the "St. Petersburg, FL" regional office and choose the schedule veterans tab and click on one of the appeals to schedule a veteran
1. On the case details page, click the "Schedule Veteran" action from the dropdown
1. Ensure that the regional office is populated with the RO you chose on the assign hearings page
1. Finish scheduling the veteran and once complete go through the postpone workflow from the case details page to ensure that the RO is still selected


### User Facing Changes

---

<img width="695" alt="Screen Shot 2020-10-07 at 5 40 49 PM" src="https://user-images.githubusercontent.com/1298274/95391014-48079200-08c4-11eb-84ed-41e83652b75b.png">

---

<img width="695" alt="Screen Shot 2020-10-07 at 5 40 45 PM" src="https://user-images.githubusercontent.com/1298274/95391016-48079200-08c4-11eb-8b62-f18a5ba061f1.png">
